R/run-nmtran.R

```diff
@@ -0,0 +1,223 @@
+#' Interface for running `NM-TRAN` on model objects
```

Comment:
This says "NM-TRAN" with a hyphen. `print.nmtran_process` says "NMTRAN". It's worth going through and making sure this is consistent on bbr's side (even if NONMEM itself uses them interchangeably).